src/app/containers/StoryPromo/IndexAlsos/index.jsx

```diff
@@ -118,16 +127,18 @@ const IndexAlsosContainer = ({ alsoItems, script, service, dir }) => {
 const alsoItemsPropTypes = shape({
   headlines: shape({
     headline: string.isRequired,
-  }).isRequired,
+  }),
   locators: shape({
     assetUri: string.isRequired,
     cpsUrn: string,
-  }).isRequired,
+  }),
   summary: string,
   timestamp: number,
-  cpsType: string.isRequired,
-  id: string.isRequired,
+  cpsType: string,
+  id: string,
```

Comment:
Was initially concerned about removing the required prop from these values, however I'd rather this than a duplicate component with different prop types

Reply:
Andrew has made a suggestion for how to avoid this.